stable/restore/templates/_helpers.tpl

```diff
@@ -133,14 +134,18 @@ Import ENV vars from configMaps
    You Have Been Warned.
 */}}
 {{- define "restore.envFrom" }}
-envFrom: [ configMapRef: { name: {{ .Values.database.name }}-restore } {{- range $map := .Values.restore.envFrom.configMapRef }}, configMapRef: { name: {{$map}} } {{- end }} ]
+envFrom: [ configMapRef: { name: {{ include "restore.target" . }}-restore } {{- range $map := .Values.restore.envFrom.configMapRef }}, configMapRef: { name: {{$map}} } {{- end }} ]
 {{- end -}}
 
 {{/*
 Return the restore target.
 */}}
 {{- define "restore.target" -}}
+{{- if .Values.database -}}
```

Comment:
It looks like you make it possible to inject `database.name` as `restore.target`. Is that a good idea having the fact that you removed the `database.name` from the chart values?

Reply:
This allows a user to include `database.name` in the restore chart, and have that used automatically as the `restore.target` default.

The template effectively checks whether the user has defined the `database.name` in their `values.yaml`, so it is certainly safe.

I am happy to have a discussion on whether that is a good idea...